docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -109,3 +109,24 @@ NOTE: You can also use **kubectl edit** to add the build-nudges-ref instead of p
 
 * Ensure that the existing references in the repositories of your components are correct.
 * Try running the necessary commands again, and ensure you have the correct syntax.
+
+== Customizing nudging PRs
+Nudging feature is using renovate `https://docs.renovatebot.com/` to actually do the job,
+for which you can supply your own renovate config, which will be used during nudging.
+Renovate config has to be specified in ConfigMap in your namespace.
+
+.There are two possible options:
+. Create namespace wide config in ConfigMap named `namespace-wide-nudging-renovate-config` which will be used for all nudged components in your namespace,
```

Comment:
Do Konflux users have the permissions to create a ConfigMap in their workspace? If so, what role do they need?

Reply:
I can create it in my own namespace, and from UI I can see that it is granted by role:
appstudio-admin-user-actions

Reply:
yeah seems like namespace admins can only create it, which is fine I'd say

Reply:
mmm I can add it, but it depends on infra/workspace configuration ....